hadoop-project/pom.xml

```diff
@@ -81,8 +81,8 @@
     <httpcore.version>4.4.13</httpcore.version>
 
     <!-- SLF4J/LOG4J version -->
-    <slf4j.version>1.7.30</slf4j.version>
-    <log4j.version>1.2.17</log4j.version>
+    <slf4j.version>1.7.35</slf4j.version>
```

Comment:
1.7.36 is out?

Reply:
The diff from 1.7.35 looks safe and good to upgrade here. I bumped the version.

> 2022-02-08 - Release of SLF4J 1.7.36
> Correct corrupt "Export-Package" declaration in MANIFEST.MF in log4j-over-slf4j module. This fixes SLF4J-541 reported by Flavio Donzé.
> 
> Starting with version 1.7.36, slf4j releases will be reproducible. By reproducible we mean that anyone checking out the code corresponding to the release version from source code repository and building that local copy, will obtain an identical binary to the published binary.